cmd/atlas/internal/mux/mux.go

```diff
@@ -0,0 +1,72 @@
+package mux
```

Comment:
Move it to atlas as well. Or, use a different naming such as `mux.URL`, `openurl.Mux`,

Reply:
moved to atlas